react/PickupPointsModal.js

```diff
@@ -265,25 +275,23 @@ class PickupPointsModal extends Component {
               pickupOptions={pickupOptions}
               salesChannel={salesChannel}
               orderFormId={orderFormId}
-              selectedPickupPoint={selectedPickupPoint}>
+              selectedPickupPoint={selectedPickupPoint}
+            >
               <Geolocation
                 address={searchAddressWithAddressQuery}
                 askForGeolocation={askForGeolocation}
                 googleMaps={googleMaps}
                 onChangeAddress={this.handleAddressChange}
                 onChangeGeolocationState={this.handleChangeGeolocationLoading}
-                rules={rules}>
+                rules={rules}
+              >
                 <SearchArea
                   address={searchAddressWithAddressQuery}
                   mapStatus={mapStatus}
                   shouldShow={shouldShowMap}
                   isLargeScreen={isLargeScreen}
                 />
-                <ZoomControls
-                  isLargeScreen={isLargeScreen}
-                  mapStatus={mapStatus}
-                  shouldShow={shouldShowMap}
-                />
+                <div id="controls-wrapper" />
```

Comment:
yep, I use it as a target for a react portal, because I needed to move the `ZoomControls` component to be a child of `Map` but the actual content of the zoom component must be placed in this div position

not the best solution, not proud of this 😂

Reply:
ah I see, maybe add a lil comment right above it

Reply:
done :-)